src/types/components.ts

```diff
@@ -0,0 +1,18 @@
+import { FileObject } from "./aws";
+
+//selector
+export type ValKey = string | number;
+export type OptionType<V> = { label: string; value: V };
+
+//dropzone
+export type Asset = {
```

Comment:
The more I'm looking at it, the more I get the feeling of how generic of a name `Asset` is. Maybe a better one would be `FileDropzoneAsset`?
It'd be a small change, but would go a long way of making it obvious what its purpose is.

Reply:
I agree with this ser, I'll include this change in #2467